lib/model/promotion.rb

```diff
@@ -39,6 +44,7 @@ def valid!
       true
     end
 
+    # CON: Could beneft from a linter such as rubocop, making sure the indentation is consistent
```

Comment:
I'm actually using Rubocop with Omakase here, some prefer to have the private section of the class indented, maybe it's the default for Omakase and I haven't checked. But I'd prefer not indenting as well